superchain/configs/configs.json

```diff
@@ -493,8 +493,7 @@
           "DataAvailabilityType": "eth-da",
           "optimism": {
             "eip1559Elasticity": 6,
-            "eip1559Denominator": 50,
-            "eip1559DenominatorCanyon": 250
```

Comment:
This parameter was not present when the chain was started up, but it needs to be there now since canyon was activated.